heron/packing/tests/java/com/twitter/heron/packing/binpacking/FirstFitDecreasingPackingTest.java

```diff
@@ -81,6 +81,35 @@ private static PackingPlan getFirstFitDecreasingPackingPlanRepack(
     return packing.repack(currentPackingPlan, componentChanges);
   }
 
+  PackingPlan doScalingTest(TopologyAPI.Topology t,
```

Comment:
@billonahill This gives me a checkstyle error that that the topology hides a field because the class contains a topology member. This is the reason I used t.